src/java/picard/vcf/SortVcf.java

```diff
@@ -34,7 +34,17 @@
 @CommandLineProgramProperties(
         usage = "Sorts one or more VCF files according to the order of the contigs in the header/sequence dictionary and then by coordinate. " +
         "Can accept an external sequence dictionary. If no external dictionary is supplied, multiple inputs' headers must have " +
-        "the same sequence dictionaries. Multiple inputs must have the same sample names (in order)\n",
+        "the same sequence dictionaries. Multiple inputs must have the same sample names (in order)" +
```

Comment:
Missing a period; also I think you need to clarify what that "(in order)" means : do the samples need to be listed in the same order in the different inputs?